examples/fit_with_bounds.py

```diff
@@ -18,21 +18,32 @@
 init_logging(level="debug")
 np.random.seed(15816)
 # {{{ helper function(s)
-class myfitclass (object):
-    def __init__(self):
+class myfitclass (nddata):
+    def __init__(self,*args,**kwargs):
+        # copied from fitdata
         self.expression = None
-    expression = (2+2)
```

Comment:
will we still need a class variable saying expression = None? or that's taken care of inside of __init__?

Reply:
Inside init. We want an instance attribute.